packages/services/f-http/src/createClient.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import defaultOptions from './defaultOptions';
+
+let _configuration = null;
+let _axiosInstance = null;
+
+/**
+    * Get a resource
+    * @param {string} resource - The resource to post (URL)
+    * @return {object} - Returns data from response
+*/
+const get = async resource => {
```

Comment:
I think we should have the ability to pass headers in here as well.

Reply:
Fair - Implemented